source/tit/core/sys_utils.hpp

```diff
@@ -21,15 +25,40 @@ using atexit_callback_t = void (*)();
 /// Register a function to be called at exit.
 void checked_atexit(atexit_callback_t callback) noexcept;
 
+/// Exit code.
+enum class ExitCode : uint8_t {
+  success = 0, ///< Success.
+  failure = 1, ///< Failure.
+};
+
 /// Exit from the current process.
 [[noreturn]]
-void exit(int exit_code) noexcept;
+void exit(ExitCode exit_code) noexcept;
 
 /// Fast-exit from the current process.
 ///
 /// @note No at-exit callbacks are triggered, except for coverage report.
 [[noreturn]]
-void fast_exit(int exit_code) noexcept;
+void fast_exit(ExitCode exit_code) noexcept;
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+/// Stack trace.
+class Stacktrace final : public boost::stacktrace::stacktrace {
```

Comment:
I am not sure this class should be here. Maybe we shall move it to `utils.hpp` inside the `Std` namespace.